company/okrs/2020_q1.md

```diff
@@ -0,0 +1,114 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. CEO KR: [$N][N] net new ARR
+   1. CEO KR: Sales leadership owns revenue targets
+   1. CEO KR: Sales, engineering, and product have a successful process for prioritization
+1. **CEO: Aspirational, efficient all-remote team**
```

Comment:
Can you add a KR to make our handbook fast to load. It still isn't. I suspect it is slow after a deploy, which is not infrequent.

Reply:
@sqs

Reply:
Yeah, this is really annoying. I investigated a bit and filed https://github.com/sourcegraph/about/issues/499. If I can't close that issue by the time OKRs are ready, then I will incorporate that into an OKR.